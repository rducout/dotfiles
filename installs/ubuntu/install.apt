google-chrome-stable
chrome-gnome-shell
nitrogen
tilix
conky
