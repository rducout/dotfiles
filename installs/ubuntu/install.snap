code
slack
inkscape
gimp
vlc
skype
